src/exportMap/patternCapture.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Traverse a pattern/identifier node, calling 'callback'
+ * for each leaf identifier.
+ * @param  {node}   pattern
+ * @param  {Function} callback
+ * @return {void}
+ */
+export function recursivePatternCapture(pattern, callback) {
```

Comment:
```suggestion
export default function recursivePatternCapture(pattern, callback) {
```

Reply:
Sounds like we want to dog-food `prefer-default-export`

- https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/prefer-default-export.md

Reply:
- https://github.com/import-js/eslint-plugin-import/pull/2991/commits/b4b1740153f293ef49982ae286fa1838a711c14d